openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -263,7 +263,25 @@ def get_libraries_for_user(user, org=None, library_type=None, text_search=None):
             Q(learning_package__description__icontains=text_search)
         )
 
-    return permissions.perms[permissions.CAN_VIEW_THIS_CONTENT_LIBRARY].filter(user, qs)
+    filtered = permissions.perms[permissions.CAN_VIEW_THIS_CONTENT_LIBRARY].filter(user, qs)
+
+    if order:
```

Comment:
nit: since we need this `order` param to just be a single field name, maybe it should be named accordingly?  e.g. `order_by_field` or `order_by_field_name`

Reply:
I see what you're saying. Do we anticipate to have different ways to order the content libraries? I feel like it would be more generic to have just `order` to accommodate the possibility of having a non-field way to order the libraries. Otherwise it might be confusing or we might need to rename the param again in the future, what do you think?

Reply:
Personally I would just keep it simple and use a list of allowed options: `["title", "created", "updated"]`

Reply:
This is meant for the discussion above right? Regarding validating whether the field is valid or not?

Reply:
Ah, yeah sorry - wrong place to reply I guess.

Reply:
I like the simplicity of this approach, I've updated it it here: [9b56378](https://github.com/openedx/edx-platform/pull/35005/commits/9b563780335836fcb68f55e3448d750fe62325b0)